### PR TITLE
Add memory.used_percent for RedisCollector

### PR DESCRIPTION
**Pull request by shortdudey123**

### CODE REVIEW

File: src/collectors/redisstat/redisstat.py [hunk 1]
```diff
@@ -285,6 +303,16 @@ def collect_instance(self, nick, host, port, unix_socket, auth):
         # server
         data = dict()
 
+        # Connect to redis and get the maxmemory config value
+        # Then calculate the % maxmemory of memory used
+        maxmemory_config = self._get_config(host, port, unix_socket, auth,
+                                            'maxmemory')
+        if maxmemory_config is not None and \
```
Comment (automatedops) [inline src/collectors/redisstat/redisstat.py]: Can simplify to `if maxmemory_config`

File: src/collectors/redisstat/redisstat.py [hunk 2]
```diff
@@ -285,6 +303,16 @@ def collect_instance(self, nick, host, port, unix_socket, auth):
...
+           'maxmemory' in maxmemory_config.keys():
```
Comment (automatedops) [inline src/collectors/redisstat/redisstat.py]: `maxmemory_config.has_key('maxmemory')`

**Comment by terencetung** [comment]
looks good to me. :shipit:

**Comment by automatedops** [comment]
👍

**Pull request merged**